battleship.py

```diff
@@ -243,6 +243,14 @@ def placeShip(data):
 Returns: None
 '''
 def clickUserBoard(data, row, col):
+    k=data["userboard"]

```

Comment:
Maintain a proper naming conventions and commit id please change to 'click user board' currently you have given by 'place ships' commit id

Reply:
Done with changes